app/dashboard/templates/external_bounties.html

```diff
@@ -0,0 +1,116 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+  </head>
+
+  <body class="interior {{active}} with-grey-background">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header dash">
+      {% include 'shared/nav_internal.html' %}
+    </div>
+    {% include 'shared/current_balance.html' with full=True %}
+
+    <div class="row no-gutter">
+      <div class="col body">
+        <div class="container" id="bounties">
+          <header class="bounties-header">
+            <div class="bounties-header__title">
+              <h2 class="title">Bounties</h2>
+            </div>
+            <div class="bounties-header__search">
+              <div class="bounties-header__info">Listing {{ bounties|length }} open bounties across the interwebs</div>
+              <div class="search_area search_area--append">
+                <input class="gc-round-text-input gc-border-black w-100" type="text" placeholder="Search Bounties" />
+                <span class="icon">{% include 'svgs/search.svg' %}</span>
+              </div>
+            </div>
+          </header>
+          <div class="main-content">
+            <aside class="aside">
+              <h3 class="sub-title">Categories</h3>
+              <div class="categories-list">
+                {% for category in categories %}
+                  <div class="checkbox">
+                    <input class="checkbox-input" name="category" type="checkbox" id="{{category}}"/>
+                    <label class="checkbox-label" for="{{category}}">{{category}}</label>
+                  </div>
+                {% endfor %}
+              </div>
+            </aside>
+            <main class="bounties-container">
+              {% for bounty in bounties %}
+                <a href="external_bounty/{{i}}" class="bounty">
```

Comment:
Just a note: Once the backend is completed for this, we'll probably want to modify these hardcoded references to use `{% url '' %}` reverse lookups in the event we change the URL path.  I wouldn't do it now since we're requesting that you comment out the url entries for the moment.